### PR TITLE
Add ApproveRules.py, MyRSEs.py, get_ff.py and dataops_ff to SiteOps

### PR DESCRIPTION
**Pull request by huangch-fnal**
ApproveRules.py -- batch approval of rules
MyRSEs.py -- list the RSEs that an account is authorized to approve quota/rules
get_ff.py -- generate tape families for rules, the output can be fed to dataops_ff
dataops_ff -- taking output from get_ff.py, generate (and/or execute) actual commands to set appropriate directories and tags. This is very Enstore, and perhaps FNAL, specific.

These scripts are provided as "examples".
The source code is the best documentation.

### CODE REVIEW

**Comment by huangch-fnal** [comment]
Moving those scripts from DMOps to SiteOps.

File: SiteOps/ApproveRules.py [hunk 1]
```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python
+
+import sys
```
Comment (ericvaandering) [inline SiteOps/ApproveRules.py:3]: Please at least make this as compatible as possible with python3 by adding `from __future__ import absolute_import, division, print_function` and adjusting the print functions.

File: SiteOps/ApproveRules.py [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+    print "    -h this message"
+    print "    -s silent mode, no output; without -a, it still prompt for action"
+    print "    -a auto mode, automatically approve the ones that can be approved"
+    print "       by default, without '-a', %s asks for confirmation"%(me)
+    print "    -v verbose mode, overridden by -s"
+    print
+    print "    ** without arguments, %s takes input from stdin"%(me)
+    print "       and gives warning if '-a' is not set"
+
+opt, args = getopt.getopt(sys.argv[1:], "f:hasv")
```
Comment (ericvaandering) [inline SiteOps/ApproveRules.py:33]: Please use argparse. getopt is super old

https://docs.python.org/3.8/library/argparse.html#module-argparse

File: SiteOps/MyRSEs.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    print "       without <account>, the current account takes place."
+
+def myrses(account=None):
+    rses = []
+    if account == None:
+        account = c.whoami()['account']
+
+    for r in c.list_rses():
+        attributes = c.list_rse_attributes(r['rse'])
+        if (attributes.has_key('quota_approvers') and attributes['quota_approvers'].find(account) != -1) \
```
Comment (ericvaandering) [inline SiteOps/MyRSEs.py:27]: Can do both of these with "in" I think. 
`if 'quota_approvers' in attributes and account in attributes['quota_approvers']`

File: SiteOps/get_ff.py [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+#
+#    the format of data is like the followings
+#
+#    dataset name is a "/" delimited path, the last term is the type
+#    block name is a dataset name with #<block>
+#
+import sys
+import os
+
+from rucio.client.didclient import DIDClient
```
Comment (ericvaandering) [inline SiteOps/get_ff.py:19]: No need to import two different clients. Client will do it.

File: SiteOps/ApproveRules.py (hunk 1 above)
Comment (huangch-fnal) [reply SiteOps/ApproveRules.py:4]: Added from __future__ import absolute_import, division, print_function and adjusted print functions.

File: SiteOps/ApproveRules.py (hunk 2 above)
Comment (huangch-fnal) [reply SiteOps/ApproveRules.py]: Yes. getopt is old and I am from old school. Replaced getopt with argparse. :-)

File: SiteOps/MyRSEs.py (hunk 3 above)
Comment (huangch-fnal) [reply SiteOps/MyRSEs.py]: Changed accordingly.

File: SiteOps/get_ff.py (hunk 4 above)
Comment (huangch-fnal) [reply SiteOps/get_ff.py]: use rucio.client.Client()

**Pull request merged**